code/controllers/ErrorPageControllerExtension.php

```diff
@@ -9,7 +9,8 @@
 class ErrorPageControllerExtension extends Extension {
 
 	public function onBeforeHTTPError($statusCode, $request) {
-		$response = ErrorPage::response_for($statusCode);
+		// Don't use the HTML response for media requests
+		$response = $request->isMedia() ? null : ErrorPage::response_for($statusCode);
```

Comment:
Ah well you would think so but see https://github.com/silverstripe/silverstripe-framework/issues/4149 :)